### PR TITLE
Perceptual Loss Function

### LINKED ISSUE #2: SRGAN-Loss-Function
Goal: Create loss function for SRGAN
How: Create custom loss function by combining existing PyTorch Loss Functions

Design Inspiration: https://github.com/Akettle44/SenNetworks/blob/main/transformers.ipynb
Explore best way to verify loss functions in PyTorch.

Acceptance Criteria: Implement Loss Function in custom class. Create unit test to verify that loss is computed correctly.

### PR DESCRIPTION
**Pull request by Akettle44**
This PR implements the perceptual loss function from SRGAN in PyTorch using VGG.

### CODE REVIEW

**Comment by StevenChang5** [comment]
LGTM, all tests passed.
